topics/epigenetics/tutorials/atac-seq/tutorial.md

```diff
@@ -95,6 +96,7 @@ We first need to download the reads (fastqs) as well as other annotation files.
 >    - *"group"*: `Genes and Gene Prediction`
```

Comment:
The issue with using this tool is that it never contains the name of the gene, only the transcript id..